pyproject.toml

```diff
@@ -37,7 +37,7 @@ sphinx-rtd-theme = "0.5.0"
 tomlkit = "^0.7.0"
 tox = "^3.15.1"
 pre-commit = "^2.17.0"
-pydantic = ">=1.8.0"
+pydantic = ">=1.8.0,<2"
```

Comment:
This should probably be a separate commit?

Reply:
This has now been moved to a separate commit